docs/protocol/introduction.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 1
+title: "Introduction"
+---
+
+import Tooltip from '@site/src/components/Tooltip';
+
+# Introduction
+
+## What is Nitrolite?
+
+Nitrolite is a chain-agnostic <Tooltip content="A secure communication pathway between participants that locks funds in an on-chain smart contract while enabling off-chain state updates.">state channel</Tooltip> protocol that enables off-chain interactions between <Tooltip content="An entity (identified by a wallet address) that is part of a channel. Typically includes a Creator (user) and a clearnode or a Clearnode Operator.">participants</Tooltip> with minimal on-chain operations. The protocol forms a unified virtual ledger (called "clearnet") for applications to escrow funds while being fully abstracted from the underlying blockchain.
+
+The initial version of Nitrolite is EVM-compatible and designed for deployment on Layer 1 and Layer 2 Ethereum networks.
```

Comment:
I would avoid specifying that as on-chain protocol specification is chain-agnostic, although only represented in Solidity as a language of reference.